src/plone/restapi/serializer/blocks.py

```diff
@@ -193,3 +198,84 @@ class SlateTableBlockSerializer(SlateTableBlockSerializerBase):
 @adapter(IPloneSiteRoot, IBrowserRequest)
 class SlateTableBlockSerializerRoot(SlateTableBlockSerializerBase):
     """Serializer for site root"""
+
+
+class TeaserBlockSerializerBase:
+    order = 0
```

Comment:
Note to self: test whether this also works for teasers inside a grid. The resolveuid transform at the top level might happen too early and resolve the URLs in nested blocks before this runs there.